specification/applicationinsights/resource-manager/readme.md

```diff
@@ -228,7 +228,6 @@ These settings apply only when `--tag=package-2021-11-01` is specified on the co
 
 ``` yaml $(tag) == 'package-2021-11-01'
 input-file:
-  - Microsoft.Insights/stable/2015-05-01/aiOperations_API.json
```

Comment:
Can we just edit an existing package (which was possibly already published as SDKs)? Or are we supposed to create a new one?

Reply:
Also, I just received a complaint from the customer that the components_API.json in this package is too old. Can we use this chance to upgrade that as well?